README.md

```diff
@@ -38,39 +38,60 @@
 
 A detailed changelog is available in the [releases](https://github.com/angular-schule/angular-cli-ghpages/releases) section.
 
-**⚠️ BREAKING CHANGE (v1)**
+**⚠️ BREAKING CHANGE (v2)**
 
-Starting with version 1 the option `--configuration` was renamed to `--build-target`.
+The internal build of Angular has changed with Angular 17.
+Unfortunately, there are now a lot of different _build targets_ and builders.
+We will try to guess the correct build target, based on the usual conventions to name them.
+The conventions are shown below, try to specify the build target more and more explicitely until the project compiles.
 
-BEFORE (_does not work_):
+In the following example, your app is called `test` and you want to deploy the `production` build.
 
-```
-ng deploy --configuration=test
+```bash
+ng deploy
 ```
 
-NOW:
+If this doesn't work, try this:
 
-```
+```bash
 ng deploy --build-target=test
 ```
 
-If you use the old syntax, you will probably receive the following error:
+If this doesn't work, try this:
+
+```bash
+ng deploy --build-target=test:build:production
+```
+
+You can also and modify your `angular.json` to archive the same:
 
-> An unhandled exception occurred: Configuration 'test' is not set in the workspace.
+```json
+{
+  "deploy": {
+    "builder": "angular-cli-ghpages:deploy",
+    "options": {
+      "buildTarget": "test:build:production"
+    }
+  }
+}
+```
 
-<br>
+For your convenience, you can also use `prerenderTarget` (which adds the suffix `:prerender:production`).
+There is no support for `universalBuildTarget` or `serverTarget` because Github pages only supports static assets!
```

Comment:
```suggestion
There is no support for `universalBuildTarget` or `serverTarget` because Github Pages only supports static assets and no Server-Side Rendering!
```